index.bs

```diff
@@ -2630,8 +2630,9 @@ Council Chairing</h5>
 <h5 id=council-convention>
 Convening the Council</h5>
 
-	Upon appointment of the [=W3C Council Chair=]
-	and delivery of the [=Team=]’s report,
+	Once [=dismissal=], [=renunciation=], and
```

Comment:
Minor ease-of-reading suggestion, no problem if it isn't taken. Just because I think the use of "Once" like this is _slightly_ archaic and also faintly suggestive of "one time only" whereas the state of being convened lasts until the council has concluded.

```suggestion
	When [=dismissal=], [=renunciation=], and
```